tests/ui/pages/desktop/base.py

```diff
@@ -26,13 +35,70 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    @property
+    def amo_logo_title(self):
+        return self.selenium.find_element(*self._amo_logo_link_locator).get_attribute('title')
```

Comment:
I don't personally think this is a good thing to check directly. Perhaps the page could provide an indirect `current_application` property that determines this based on the logo alt text, page title, or logo title as you have here.